META6.json

```diff
@@ -1,6 +1,6 @@
 {
   "depends": [
-    "CSS::Grammar:ver(v0.3.3+)",
+    "CSS::Grammar:ver<v0.3.3+>",
```

Comment:
This should probably be `CSS::Grammar:ver<0.3.3+>`. In the previous code it includes the `v` because its being used as a version literal (since it is inside the parens its actual code).